app/src/main/java/com/github/palFinderTeam/palfinder/meetups/fragments/CriterionsFragment.kt

```diff
@@ -54,14 +60,9 @@ class CriterionsFragment(val viewModel: MeetUpCreationViewModel) : DialogFragmen
             CriterionGender.ALL -> sexGroup.check(R.id.radioMaleAndFemale)
         }
 
-
-
-        // Add to layout
-        val layout = v.findViewById(R.id.ageSeekBar) as FrameLayout
-        layout.addView(rangeSeekBar)
-
         val button: Button = v.findViewById(R.id.criterionButtonDone)
 
+        // Set gender, or as Antoine would call it, sex.

```

Comment:
Funny add ;) But should remove and keep only "Set gender"

Reply:
You're being too serious! Haha
I will remove it in the next commit :)